### PR TITLE
 MAGEDOC-3542-ui-terminology

### PR DESCRIPTION
**Pull request by ccondra**
image

### CODE REVIEW

File: src/marketing/gift-registries.md [hunk 1]
```diff
@@ -5,8 +5,13 @@ title: Gift Registries
...
 
-![Example storefront - gift registry]({% link images/images-ee/storefront-gift-registry-create-baby-info.png %}){: .zoom}
-_Create Gift Registry in Customer Dashboard_
+<!--{% if "Default.EE Only" contains site.edition %}-->
+![]({% link images/images-ee/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.B2B Only" contains site.edition %}-->
+![]({% link images/images-b2b/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% endif %}-->
+_Creating a Baby Gift Registry_
```
Comment (leslietilling) [inline src/marketing/gift-registries.md]: We don't want to get rid of the alt text here. We are in the process of putting them in. The `[Example storefront - gift registry]` is basically meta data that affects our SEO. It does not necessarily match the caption. Going forward we want all images to have alt text.

File: src/marketing/gift-registry-information.md [hunk 2]
```diff
@@ -3,9 +3,14 @@ conditions: Default.EE-B2B
...
-Customers can create and manage gift registries from their accounts. The account dashboard includes the information that is needed for the type of registry.
+Customers can create and manage gift registries from their accounts. All of the information that is associated with each type of registry is available from the customer's account. 
 
-![Example storefront - gift registry information]({% link images/images-ee/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% if "Default.EE Only" contains site.edition %}-->
+![]({% link images/images-ee/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.B2B Only" contains site.edition %}-->
+![]({% link images/images-b2b/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% endif %}-->
```
Comment (leslietilling) [inline src/marketing/gift-registry-information.md]: Same as above. We want to keep the alt text.

File: src/customers/account-dashboard-my-wish-list.md [hunk 3]
```diff
@@ -39,10 +55,38 @@ _My Wish List_
...
     _Shared Wish List Email_
+<!--{% if "Default.EE-B2B" contains site.edition %}-->
+
+## To create a new wish list:
+
+1. In the upper-right corner, click **Create New Wish List**.
+   
+1. Enter the Wish List Name. 
+    
+1. If you want others to be able to see your wish list, mark the **Public Wish List** checkbox,
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md]: `select` instead of `mark`

Comment (ccondra) [reply src/customers/account-dashboard-my-wish-list.md]: Fixed

File: src/customers/account-dashboard-my-downloadable-products.md [hunk 4]
```diff
@@ -2,25 +2,28 @@
...
 
-1. In your account dashboard, choose **My Downloadable Products**.
+1. In your account, choose **My Downloadable Products**.
 
 1. Find the order in the list and click the _download_ link after the title.
 
 1. In the lower-right corner of the download window, click the _download_ icon.
 
-1. Look for the name of the download file to appear in the lower-left corner of the window and save the file.
+1. Look for the name of the download file to appear in the lower-left corner of the window and save the file. On the Mac, display the menu in the lower-right corner, and choose **Download**.
```
[the remark this answers is not in the document]
Comment (ccondra) [reply src/customers/account-dashboard-my-downloadable-products.md]: Fixed.

File: src/customers/account-dashboard-my-invitations.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+The My Invitations section of the customer account lists all [invitations]({% link marketing/invitations.md %}) sent by the customer. Customers can send invitations to friends and family for store events, gift registries, wish lists, and so on. 
+
+<!--{% if "Default.EE ONLY" contains site.edition %}-->
+![]({% link images/images-ee/account-dashboard-my-invitations.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.B2B Only" contains site.edition %}-->
+![]({% link images/images-b2b/customer-account-dashboard-my-invitations.png %}){: .zoom}
+<!--{% endif %}-->
+_[My Invitations]({% link marketing/invitations.md %})_
```
[the remark this answers is not in the document]
Comment (ccondra) [reply src/customers/account-dashboard-my-invitations.md]: Fixed.

**Comment by ccondra** [comment]
Running tests

**Comment by ccondra** [comment]
Running tests

**Comment by ccondra** [comment]
Complete

**Pull request merged**